tools/installers/install_torch.sh

```diff
@@ -154,7 +154,7 @@ if ! python -c "import packaging.version" &> /dev/null; then
     python3 -m pip install packaging
 fi
 
-if $(pytorch_plus 1.12.2); then
+if $(pytorch_plus 1.13.2); then
```

Comment:
Is it fine if we extend support to 1.13.1 ?

Reply:
Sorry, I didn't check this thread. I have created a PR.